src/packet_analysis/protocol/ppp_serial/CMakeLists.txt

```diff
@@ -1,8 +1,7 @@
-
-include(ZeekPlugin)
-
-include_directories(BEFORE ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
-
-zeek_plugin_begin(PacketAnalyzer PPPSerial)
-zeek_plugin_cc(PPPSerial.cc Plugin.cc)
-zeek_plugin_end()
+zeek_add_plugin(
+    Zeek
```

Comment:
Should this be `PacketAnalyzer` instead of `Zeek`?

Reply:
Fixed.